MusicNotationKit/MusicNotationKit/Measure.swift

```diff
@@ -68,6 +68,23 @@ extension Measure: NotesHolder {
 	
 }
 
+// Debug extensions
+extension Measure: CustomDebugStringConvertible {
+	public var debugDescription: String {
+		var notes = [String]();
+		for note in self.notes {
+			notes.append(String(note))
+		}
+		let notesString = notes.joinWithSeparator(", ")
+		
+		let result:String = String(format: "|%@: %@|",
+			String(self.timeSignature),
+		    notesString)
+
+		return result
+	}
```

Comment:
Using the comments made in the last file, rewrite this similarly. I won't give it to you so you can practice applying the techniques.